print-udp.c

```diff
@@ -646,6 +646,8 @@ udp_print(netdissect_options *ndo, register const u_char *bp, u_int length,
 			geneve_print(ndo, (const u_char *)(up + 1), length);
 		else if (IS_SRC_OR_DST_PORT(LISP_CONTROL_PORT))
 			lisp_print(ndo, (const u_char *)(up + 1), length);
+		else if (dport == GUE_PORT)
```

Comment:
I am not sure which kind of packet exactly but generally packets in tunnels tend to go both ways. Unless, of course, in this protocol the return traffic uses a completely unrelated source and destination ports. I did not study the spec so well.